test/wpt/status/console.json

```diff
@@ -1,5 +1,5 @@
 {
   "idlharness.any.js": {
-    "fail": ".table, .dir and .timeLog parameter lengths are wrong"
+    "skip": ".table, .dir and .timeLog parameter lengths are wrong"
```

Comment:
Why is this changed to `skip`?

Reply:
Because that's effectively what the current behaviour does. Tests run but their failure or pass does nothing with the script outcome.

Reply:
I wouldn't mind expanding these skips to be the failed test names like its done with webcrypto. I did so for all files that had partial passes, but when everything fails, changing old fail to new skip has the same outcome.

Reply:
I checked locally and indeed it now just ignores unexpected passes - I probably got the impression that it should fail from ancient behaviors, sorry about the confusion. I still think that we should use `fail` for tests where all the test cases fail though (and it is fine to use one string to describe the reason for everything)

Reply:
> I still think that we should use fail for tests where all the test cases fail though (and it is fine to use one string to describe the reason for everything)

There are seldomly wpt files where ALL the test cases fail. And treating those as skip with a reason vs a fail with a reason does not make a difference. If anything, marking them as skip makes it a clear we either don't care and/or need to investigate.

Having one fail reason which even swallows regressions is not helpful. 

I'll refactor the fails that lost their "one reason" to have a note in the json file. 

The rest I believe is valid and puts us in a better place where we can

- identify regressions
- cross off fixes in an iterative manner
- show how a fix affects the wpt state (see webcrypto)

Reply:
FYI, see these [CI failures](https://ci.nodejs.org/job/node-test-pull-request/45938/). console results seem to depend on the platform. I'll have to either skip the whole file or flake the individual ones.